scripts/configure-vaults.js

```diff
@@ -64,11 +69,16 @@ async function main() {
     const SHADOW_WS_STS_POOL = "0xde861c8fc9ab78fe00490c5a38813d26e2d09c95"
     const SHADOW_WS_WETH_POOL = "0xb6d9b069f6b96a507243d501d1a23b3fccfc85d3"
 
+    const ICHI_EQUALIZER_WS_USDC_WANT = "0x238C26DDEa068190Fe715b095463B82796F0058B";
```

Comment:
I went through the ABI and code deployed on Base/Sonic and it looks like this is not correct address, in case of ICHI, `want` should point to ICHIVault.

Based on tx: https://sonicscan.org/tx/0x344f3b80a3e4e758260ce7897ed79b87e44d25361631fce17973cf416ef988b1
It points to 0x238C26DDEa068190Fe715b095463B82796F0058B